sudoku/script.js

```diff
@@ -39,3 +52,92 @@ btn.addEventListener('click', () => {
   }
 });
 btn.addEventListener('click', addRandom);
+
+let checkColumnsArr = [];
+function checkColumns(matrix) {
+  let temp, z;
+  for (let i = 0; i < matrix.length; i++) {
+    checkColumnsArr.splice(0, z);
+    for (let j = 1; j < matrix[i].length; j++) {
+      temp = matrix[0][i];
+      checkColumnsArr.push(matrix[j][i]);
+      z = matrix[j].length - 1;
+    }
+    let x = new Set(checkColumnsArr);
+    let y = temp;
+    for (let props of x) {
+      if (props === y || x.size != z) {
+        answers.push(false);
+        break;
+      } else continue;
+    }
+  }
+}
+
+let checkRowsArr = [];
+function checkRows(matrix) {
+  let temp, z;
+  for (let i = 0; i < matrix.length; i++) {
+    checkRowsArr.splice(0, z);
+    for (let j = 1; j < matrix[i].length; j++) {
+      temp = matrix[i][0];
+      checkRowsArr.push(matrix[i][j]);
+      z = matrix[i].length - 1;
+    }
+    let x = new Set(checkRowsArr);
+    let y = temp;
+    for (let props of x) {
+      if (props === y || x.size != z) {
+        answers.push(false);
+        break;
+      } else continue;
```

Comment:
No need for this, anyways you don't have anything after continue
```suggestion
      } 
```